### DISCUSSION TITLE
Multi -Tenancy of Security Onion

### DISCUSSION THREAD

**Discussion by constantinos9696** [Security-Onion-Solutions/securityonion#5136] [General] [answered]
Is it possible to create multi tenancy in security onion?

We need it for our clients ASAP in order to integrate them on security onion,

Thank you in advance.

**Comment by TOoSmOotH** [comment, marked as answer]
There is no multi tenancy in Security Onion.

**Reply by constantinos9696** [reply]
Thank you very much.
